doc/classes/EditorInterface.xml

```diff
@@ -256,18 +256,24 @@
 				Restarts the editor. This closes the editor and then opens the same project. If [param save] is [code]true[/code], the project will be saved before restarting.
 			</description>
 		</method>
+		<method name="save_all_scenes">
+			<return type="void" />
+			<description>
+				Saves all opened scenes in the editor.
+			</description>
+		</method>
 		<method name="save_scene">
 			<return type="int" enum="Error" />
 			<description>
-				Saves the scene. Returns either [constant OK] or [constant ERR_CANT_CREATE].
+				Saves the current edited scene. Returns either [constant OK] or [constant ERR_CANT_CREATE].
```

Comment:
```suggestion
				Saves the currently active scene. Returns either [constant OK] or [constant ERR_CANT_CREATE].
```
As above in `play_current_scene`

Reply:
This was fixed, so I'll resolve the conversation